packages/client/src/workflow-options.ts

```diff
@@ -39,6 +39,10 @@ export interface WorkflowOptions extends CommonWorkflowOptions {
   followRuns?: boolean;
 }
 
+export interface WorkflowUpdateOptions {
+  readonly id?: string;
```

Comment:
Please qualify this id, something like `updateRequestId`

Reply:
It's not a request ID, I'm fine just calling this `id` TBH, but also okay if we want to use `updateId`.

Reply:
My concern here is that since we are not making this mandatory, most TS users will probably not be familiar with the fact that updates accept an ID, and what that ID means. If we simply call it `id`, one could easily mistaken that to be something else (ie. workflow id, run id, or whatever else).